Website installation and test instructions reference incorrect venv name (st_venv instead of shim_venv)

### Environment details
I followed the [installation instructions](https://shimming-toolbox.org/en/latest/getting_started/installation_st/mac.html)
Project version: Newest version as of 2021-09-21 (see issue 302 https://github.com/shimming-toolbox/shimming-toolbox/issues/302 for st_version request)
Operating System and version: Mac OS 10.15.7
Python version: 3.7.10

### Bug Description

The testing instructions on the website (https://shimming-toolbox.org/en/latest/getting_started/installation_st/mac.html#test-the-installation-optional) refer to a virtual environment named st_venv. However, the current install script creates a virtual environment called shim_venv. This should be fixed to avoid user confusion.
The same issue is present on the Linux install instructions, but I havent tested if they cause an error.

Well this is getting weirder.
1, Removal was successful.
2, Make install failed with:
```
(base) danielpapp@Daniels-MacBook-Pro ~ % cd shimming-toolbox
make install
make: *** No rule to make target `install'.  Stop. 
```
Contents on the /shimming-toolbox directory, in case this is helpful:
```
(base) danielpapp@Daniels-MacBook-Pro shimming-toolbox % ls
CODE_OF_CONDUCT.md			pytest.ini
LICENSE					python
README.md				readthedocs.yml
__pycache__				setup.py
bin					shimming-toolbox
config					shimming-toolbox-0.1-beta
conftest.py				shimming-toolbox-0.1-beta.tar.gz
dcm2bids.json				shimmingtoolbox
docs					shimmingtoolbox.egg-info
examples				test
fsleyes-plugin-shimming-toolbox		testing_data
```

3, So I deleted the shimming-toolbox directory and re-cloned from git again
`git clone https://github.com/shimming-toolbox/shimming-toolbox.git`
4, And now it installed fine, and used the name st_venv:
`Beginning shimming-toolbox install in /Users/danielpapp/shimming-toolbox/python/envs/st_venv`
5,  However, the venv list is now:
```
base                  *  /Users/danielpapp/miniconda
dcm2bids                 /Users/danielpapp/miniconda/envs/dcm2bids
                         /usr/local/fsl/fslpython
                         /usr/local/fsl/fslpython/envs/fslpython
```
6, Which allows st_ commands to be called from (base):
```
(base) danielpapp@Daniels-MacBook-Pro ~ % st_
st_check_dependencies  st_dump_env_info       st_maths
st_dicom_to_nifti      st_image               st_prepare_fieldmap
st_download_data       st_mask                st_realtime_shim
```
7, But causes the test step to fail:
```
(base) danielpapp@Daniels-MacBook-Pro ~ % cd shimming-toolbox
source $HOME/shimming_toolbox/python/etc/profile.d/conda.sh
conda activate st_venv
pytest
source: no such file or directory: /Users/danielpapp/shimming_toolbox/python/etc/profile.d/conda.sh
Could not find conda environment: st_venv
You can list all discoverable environments with `conda info --envs`.

zsh: command not found: pytest
```


I'm pretty sure this is user error (since doing the apparently same thing twice leads to different outcomes), but I can't figure out where...

--- a/docs/source/getting_started/installation_st/linux.rst
+++ b/docs/source/getting_started/installation_st/linux.rst
@@ -41,7 +41,7 @@ Next, install ``shimming-toolbox`` using the ``Makefile``:
   cd shimming-toolbox
   make install
 
-Restart your terminal or source your terminal profile
+Restart your terminal or source your terminal profile.
 
 3. Test the Installation (optional)
 -----------------------------------
@@ -50,7 +50,7 @@ Comprehensive Test
 ~~~~~~~~~~~~~~~~~~
 
 To run the entire testing suite, run ``pytest`` from the
-**cloned** directory:
+**cloned** shimming-toolbox directory:
 
 .. code:: bash
 

--- a/docs/source/getting_started/installation_st/linux.rst
+++ b/docs/source/getting_started/installation_st/linux.rst
@@ -41,6 +41,7 @@ Next, install ``shimming-toolbox`` using the ``Makefile``:
   cd shimming-toolbox
   make install
 
+Restart your terminal or source your terminal profile
 
 3. Test the Installation (optional)
 -----------------------------------
@@ -49,12 +50,12 @@ Comprehensive Test
 ~~~~~~~~~~~~~~~~~~
 
 To run the entire testing suite, run ``pytest`` from the
-shimming-toolbox directory:
+**cloned** directory:
 
 .. code:: bash
 
   cd shimming-toolbox
-  source $HOME/shimming_toolbox/python/etc/profile.d/conda.sh
+  source $HOME/shimming-toolbox/python/etc/profile.d/conda.sh
   conda activate st_venv
   pytest
 

--- a/docs/source/getting_started/installation_st/mac.rst
+++ b/docs/source/getting_started/installation_st/mac.rst
@@ -29,7 +29,7 @@ Next, install ``shimming-toolbox`` using the ``Makefile``:
   cd shimming-toolbox
   make install
 
-Restart your terminal or source your terminal profile
+Restart your terminal or source your terminal profile.
 
 3. Test the Installation (optional)
 -----------------------------------
@@ -38,7 +38,7 @@ Comprehensive Test
 ~~~~~~~~~~~~~~~~~~
 
 To run the entire testing suite, run ``pytest`` from the
-**cloned** directory:
+**cloned** shimming-toolbox directory:
 
 .. code:: bash
 

--- a/docs/source/getting_started/installation_st/mac.rst
+++ b/docs/source/getting_started/installation_st/mac.rst
@@ -29,6 +29,7 @@ Next, install ``shimming-toolbox`` using the ``Makefile``:
   cd shimming-toolbox
   make install
 
+Restart your terminal or source your terminal profile
 
 3. Test the Installation (optional)
 -----------------------------------
@@ -37,12 +38,12 @@ Comprehensive Test
 ~~~~~~~~~~~~~~~~~~
 
 To run the entire testing suite, run ``pytest`` from the
-shimming-toolbox directory:
+**cloned** directory:
 
 .. code:: bash
 
  cd shimming-toolbox
- source $HOME/shimming_toolbox/python/etc/profile.d/conda.sh
+ source $HOME/shimming-toolbox/python/etc/profile.d/conda.sh
  conda activate st_venv
  pytest
 

